arm-redis/2016-04-01/swagger/redis.json

```diff
@@ -545,6 +545,210 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Cache/Redis/{name}/firewallRules/": {
+      "get": {
+        "tags": [
+          "Redis",
+          "FirewallRules"
+        ],
+        "operationId": "RedisFirewallRules_List",
+        "description": "Gets all firewall rules in the specified redis cache.",
+        "x-ms-examples": { 
+          "RedisCacheFirewallRulesList": { "$ref": "../examples/RedisCacheFirewallRulesList.json" } 
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "name",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the Redis cache."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully got the current rules",
+            "schema": {
+              "$ref": "#/definitions/RedisFirewallRuleListResult"
```

Comment:
@TimLovellSmith - https://github.com/Azure/azure-rest-api-specs/pull/977/files#diff-785771b0c72c01e60191a5d798d6d508R1128 The model has a property named nextLink. 
If you think that the server will never return more than one server page of response and the service does not have support for nextLink property in the response body then please remove nextLink as the property from the model. 

However, you would still need to add the x-ms-pageable extension to the operation. The minor difference would be `"nextLinkName": null` in the x-ms-pageable extension. This is done explicitly to indicate AutoRest that there is no nextLink. Then AutoRest will still flatten the "value" property and return an `IEnumerable<RedisFirewallRule>` instead of `IPage<RedisFirewallRule>`